config/config.go

```diff
@@ -217,7 +216,7 @@ type ScrapeConfig struct {
 	// Indicator whether the scraped metrics should remain unmodified.
 	HonorLabels bool `yaml:"honor_labels,omitempty"`
 	// A set of query parameters with which the target is scraped.
-	Params url.Values `yaml:"params,omitempty"`
+	Params map[string]string `yaml:"params,omitempty"`
```

Comment:
Just since you asked for opinions:

Yeah, supporting only a single parameter now seems ok to me. That will still allow us to switch to a numbered scheme later without breaking anything (or maybe we'll come up with another non-breaking option given our experiences with it).

Maybe I'm wrong, but it seems it will always be quite a fringe use case to have params at all, then *multiple* of them, *and* wanting to relabel them, all at the same time.